scalableminds/webknossos
# Remove most return statements from scala code

return in scala is generally harmful, compare article at https://tpolecat.github.io/2014/05/09/return.html

This PR removes most return statements from our code
 - except those in the jzarr and datastore.storage packages (because PR #6335  does that
 - except those that I did not know how to convert without rewriting a lot of code (DataFinder, IsosurfaceService, CumsumParser)

It is recommended to hide whitespace in the diff view :)

### URL of deployed dev instance (used for testing):
- https://fewerreturnsinscala.webknossos.xyz

### Steps to test:
- create task, trace some, download + upload volume data, should still work as expected

### Issues:
- contributes to #6356 

------
